### PR TITLE
Added language preferences

### PR DESCRIPTION
**Pull request by T-e-j-k-u-m-a-r**
Hi Jeevitha, 

Please review it.

### CODE REVIEW

File: movie_names.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+1. English 
+
```
Comment (jeevithatejkumar) [inline movie_names.txt:2]: @T-e-j-k-u-m-a-r : Please add movie names based on an English language

File: movie_names.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+
+2. Kannada
```
Comment (jeevithatejkumar) [inline movie_names.txt:5]: @T-e-j-k-u-m-a-r : Please add movie names based on Kannada language

File: movie_names.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+
+
+3. Tamil
```
Comment (jeevithatejkumar) [inline movie_names.txt:9]: @T-e-j-k-u-m-a-r : Please add movie names based on Tamil language

**Comment by jeevithatejkumar** [comment]
Review completed, Merging the code. 

Regards,
Tejkumar

**Comment by T-e-j-k-u-m-a-r** [comment]
@jeevithatejkumar : Merging the code as the review is complete. 

Regards,
Tejkumar

**Pull request merged**